open-ce/utils.py

```diff
@@ -82,6 +82,30 @@ class Argument(Enum):
                                         default=DEFAULT_BUILD_TYPES,
                                         help='Comma delimited list of build types, such as "cpu" or "cuda".'))
 
+    DOCKER_BUILD = (lambda parser: parser.add_argument(
+                                        '--docker_build',
+                                        action='store_true',
+                                        help="Perform a build within a docker container. "
+                                             "NOTE: When the --docker_build flag is used, all arguments with paths "
+                                             "should be relative to the directory containing open-ce. Only files "
+                                             "within the open-ce directory and local_files will be visible at "
+                                             "build time."))
+
+    USE_LOCAL_SCRATCH = (lambda parser: parser.add_argument(
+                                        '--use_local_scratch',
+                                        action='store_true',
+                                        help="When performing a build within a container, use local scratch space "
+                                             "instead of disk space within the container. This can be useful if the "
+                                             "container has limited disk space. If `local_scratch_folder` is not "
+                                             "set, scratch space will be taken from a temporary folder, which will "
+                                             "be deleted after the job has completed."))
+
+    LOCAL_SCRATCH_FOLDER = (lambda parser: parser.add_argument(
+                                        '--local_scratch_folder',
```

Comment:
Minor nit, but you reference directories above, and now folders here.  Is there a difference in the terminology?